.gitignore

```diff
@@ -1,4 +1,6 @@
 /build/*
+/_build/*
+/Testing/*
```

Comment:
My filesystem isn't case sensitive, so this ignores all files under `testing`.   :(